frontend-tests/Desktop/Test130.cy.js

```diff
@@ -0,0 +1,161 @@
+import { utility } from "../../support/Utility"
+	
+describe('test id 130 - Automate Footer link - Landing page section', () => {
+	  beforeEach(() => {
+	    cy.visit('/en/learn')
+	  })
+
+	  it('Learn overview - Top section - Learn page content', () => {
+	    cy.get('.h2')
+	      .should('be.visible')
+	  })
+	
+	  it('Find all broken links - verify broken link on landing page', () => {
+	    cy.get('a').each(link => {
+	      if (link.prop('href'))
+	        cy.request({
+	          url: link.prop('href'),
+	          failOnStatusCode: false
+	        })
+	
+	      cy.log(link.prop('href'))
+	    })
+
+	//Sources of retirement income  
+	it('Testing all English footer link 1.) Sources of retirement income ', () => {
+        cy.visit('/')
+        cy.get('#english-button').click()
+		let language = new utility().getLanguageTabletOrMonitorScreen()        
```

Comment:
**Suggestion:** if you move your utility functions under the Cypress `support/commands.js` file and configure them to be actual cypress commands, you can clean up your code a bit.. You could then chain your cypress calls together like this:

``` javascript
it('Testing all English footer link 1.) Sources of retirement income ', () => {
  cy.visit('/')
    .get('#english-button').click()
    .getLanguageTabletOrMonitorScreen()        
    .get('.col-span-2 > :nth-child(1) > .MuiTypography-root').click({ force: true })
})
```